devtools/common/src/main/java/io/quarkus/cli/commands/writer/ZipWriter.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * 
+ */
+package io.quarkus.cli.commands.writer;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+/**
+ * @author guillaumedufour
+ *
+ */
+public class ZipWriter implements Writer {
```

Comment:
Unless I am mistaken, this isn't being used anywhere (yet). Do you plan on adding some tests for it?

Reply:
it's use by the generator it self, but i can add it in the mojo too...
https://github.com/Dufgui/quarkus-generator

Reply:
i don't know if the generator must be push inside quarkus repo or in a dedicated one

Reply:
I don't know that either :). Best take it up with @emmanuelbernard and @kenfinnigan I guess.

Reply:
If we decide not have the generator here, it's probably best to have some tests for `ZipWriter`